components/Screen.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { SafeAreaView, StyleSheet, View, ImageBackground } from "react-native";
+import Constants from "expo-constants";
+
+export default function Screen({ children, style }) {
+  return (
+    <SafeAreaView style={[styles.container, style]}>
+      <ImageBackground
+        source={require("../assets/giphy.gif")}
+        style={{ width: "100%", height: "100%", resizeMode: "cover" }}
```

Comment:
In general, using the `StyleSheet` api is preferred to using inline styles like this.